src/main/java/com/springboot/star_wars_character_popularity/app/controller/MovieController.java

```diff
@@ -0,0 +1,50 @@
+package com.springboot.star_wars_character_popularity.app.controller;
+
+import com.springboot.star_wars_character_popularity.app.model.MovieModel;
+import com.springboot.star_wars_character_popularity.app.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/movie")
+public class MovieController {
+
+    @Autowired
+    private MovieService movieService;
+
```

Comment:
No need for @Autowire. 

Try using @RequiredArgsConstructor annotation(lombok) on top of the class and remove constructor and set field as final.